### DISCUSSION TITLE
Mistake in the song?

### DISCUSSION THREAD

**Discussion by Nikv2005** [Monika-After-Story/MonikaModDev#8362] [QA - tech support + general] [answered]
In the song "Renai Circulation" Monika sings "Thanks you stars, for giving to me ". But after this in dialogue she says "The Japanese version of this song's chorus actually says something more along the lines of 'Thank you god for giving to me' for the first line"

**Comment by ThePotatoGuy** [comment, marked as answer]
I am fairly certain that we don't have that song in the mod, so that must be a submod song.

**Comment by Nikv2005** [comment]
I didn't think about it ... oops, well then everything is fine. Sorry for that.
